.travis.yml

```diff
@@ -6,12 +6,10 @@ node_js:
 cache:
   yarn: true
   directories:
-  - node_modules
+    - node_modules
 env:
   - NODE_ENV=development
   - NODE_ENV=production
-matrix:
-  fast_finish: true
```

Comment:
❓

Reply:
Yeah, shouldn't we leave this, in order to learn about build failures faster? From https://blog.travis-ci.com/2013-11-27-fast-finishing-builds:

> With fast finishing enabled, Travis CI will mark your build as finished as soon as one of two conditions are met: The only remaining jobs are allowed to fail, or a job has already failed.

We don't have any [jobs allowed to fail](https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail), so it should be safe to leave `fast_finish` enabled.

Reply:
Ah, I was under the impression it only applied when jobs were allowed to fail. Reverted.